src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,45 @@
+package mate.jdbc.model;
+
+public class Manufacturer {
+    private Long id;
+    private String name;
+    private String country;
+
+    public Manufacturer() {
```

Comment:
You also can use Loombok annotation  @AllArgsConstructor @NoArgsConstructor, or @RequiredArgsConstructor,, you can read about it here: https://projectlombok.org/features/constructor

Reply:
No parameterless constructor is always created regardless of whether you show it or not